intelmq/etc/defaults.conf

```diff
@@ -6,6 +6,7 @@
     "destination_pipeline_password": null,
     "destination_pipeline_port": 6379,
     "error_dump_message": true,
+    "error_forward_message": false,
```

Comment:
see above.
We definitely need a comment here! This can have pretty bad effects if turned on.